src/otx/core/model/entity/base.py

```diff
@@ -333,13 +334,15 @@ def __init__(
         max_num_requests: int | None = None,
         use_throughput_mode: bool = True,
         model_api_configuration: dict[str, Any] | None = None,
+        tile_config: TileConfig | None = None,
```

Comment:
Like this comment https://github.com/openvinotoolkit/training_extensions/pull/2925/files#r1499081685, please make OpenVINO IR self-contained about tiling configuration. There should be no external injection for it like OTX 1.x.

Reply:
I made changes accordginly.